detection_rules/schemas/definitions.py

```diff
@@ -17,7 +17,7 @@
 DATE_PATTERN = r'^\d{4}/\d{2}/\d{2}$'
 MATURITY_LEVELS = ['development', 'experimental', 'beta', 'production', 'deprecated']
 OS_OPTIONS = ['windows', 'linux', 'macos']
-NAME_PATTERN = r'^[a-zA-Z0-9].+?[a-zA-Z0-9()]$'
+NAME_PATTERN = r'^[a-zA-Z0-9].+?[a-zA-Z0-9\[\]()]$'
```

Comment:
Have an example of why this was needed?

Reply:
Yes, any case where rules exported from Kibana have a title similar to `Rule Name [Duplicate]` 
Example from the testing block above that has this:

[rules_export_new_terms_test_other.ndjson.txt](https://github.com/elastic/detection-rules/files/14858585/rules_export_new_terms_test_other.ndjson.txt)